vnext/ReactWindowsCore/ReactWindowsCore.vcxproj.filters

```diff
@@ -4,50 +4,91 @@
     <Filter Include="include">
       <UniqueIdentifier>{d58d8a95-1e57-49ea-a4f3-c0859b7f649d}</UniqueIdentifier>
     </Filter>
-    <Filter Include="Modules">
-      <UniqueIdentifier>{81319f88-192e-40cf-bc1c-76a84bf37ccd}</UniqueIdentifier>
+    <Filter Include="Header Files">
+      <UniqueIdentifier>{32892a40-82db-4c1b-9390-5893f9ddbb0b}</UniqueIdentifier>
     </Filter>
-    <Filter Include="Chakra">
-      <UniqueIdentifier>{1c2caa01-d0f4-4fb9-a956-c8e47cbee437}</UniqueIdentifier>
+    <Filter Include="Source Files">
```

Comment:
>Source Files [](start = 21, length = 12)

(nit) "Sources"

Reply:
I guess that's because "Source Files" is vs default naming convention?

---
In reply to: [299114282](https://github.com/microsoft/react-native-windows/pull/2708#discussion_r299114282) [](ancestors = 299114282)

Reply:
Exactly